jennchenn/uwexplore
# Style components

## Implementation description

- Apologies for the fat PR
- I tried to commit by section so it would be easier to go through?
- Added styling to entire website
- Also resolves BREAD-65, BREAD-63, BREAD-96

**General**
- Added favicon and logo & changed the site title
- Changed name to uwexplore everywhere (placeholder?)
- Changed the size of the navbar and how the height of the left/right sections is calculated
- Added prettier scrollbars for the left/right sections, and within the search cards (for the table, if needed)
- For the custom Button and TextField components, I added a general prop (smtn like [key: string]: any) which allows us to access props from the MUI component (things like OnClick events)

**Calendar**
- Styled events (text should break off into "..." on overflow now) and modal
- Events will no longer overlap each other on the calendar
- Updated header and time gutter styles, and removed unecessary borders everywhere
- Added conditional styling so that the calendar should reduce its height on smaller screens, and have slightly smaller font sizes

**Search**
- Styled filtering menu and cards
- Updated search bar to use custom TextField component
- Added nicer spacing between cards/headings/menus
- If card names are too long, they should break off into "..."

**CEAB**
- Made this section look more like the figma 



## Steps to test

1. verify some of the changes above?
2. just click around/resize screen and see if anything breaks/looks weird or if there is something that could look better in general



## What should reviewers focus on?

- One known styling issue: in the CEAB section, after choosing a term in the "Term" dropdown, the word "Term" gets lost in the background since it is white. 
- Also, I realize I chose the wrong background colour for the search bar, will also be fixed
- I will work on this a bit more in the next couple of days to clean up the styles so they aren't all inline! But functionally, everything should stay the same. 
- Should we remove the option/button to collapse the search section? Is this something we still wanted in the final project
